snuba/datasets/entities/sessions.py

```diff
@@ -218,10 +218,6 @@ def select_storage(
             },
         )
 
-        allow_subhour_sessions = state.get_config("allow_subhour_sessions", 0)
```

Comment:
What's the impact of this? Is there anyone for whom this is not already available?

Reply:
This was added just as a failsafe back then. It was never turned on in production, but the metrics that are being tracked in the lines above gave some insight into which storage would have been chosen.